inbm/dispatcher-agent/fpm-template/usr/share/dispatcher-agent/manifest_schema.xsd

```diff
@@ -460,6 +460,7 @@
                 <xs:element name="fetch" type="URLMax700Chars" minOccurs="0" maxOccurs="1"/>
                 <xs:element name="file" type="Max50Chars" minOccurs="0" maxOccurs="1"/>
                 <xs:element name="version" type="MinDecimalZero" minOccurs="0" maxOccurs="1"/>
+                <xs:element name="signature" type="SigMax1000Chars" minOccurs="0" maxOccurs="1"/>
```

Comment:
Validate that the `signature` element in the XSD schema is optional but, if present, must conform to the expected format. [important]